File: 02_ProgrammingFundamentals/_03_Arrays/_02_Exercise/_09_ArrayModifier.java
package _02_ProgrammingFundamentals._03_Arrays._02_Exercise;

import java.util.Arrays;
import java.util.Scanner;
import java.util.stream.Collectors;

public class _09_ArrayModifier {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input values
        int[] array = Arrays.stream(scan.nextLine().split("\\s+")).mapToInt(Integer::parseInt).toArray();
        String input = scan.nextLine();

        // Accept commands and modify the array
        while (!input.equals("end")) {
            String[] commands = input.split("\\s+");
            String command = commands[0];

            if (command.equals("swap")) {
                int firstNumber = Integer.parseInt(commands[1]);
                int secondNumber = Integer.parseInt(commands[2]);
                int swap = array[firstNumber];

                array[firstNumber] = array[secondNumber];
                array[secondNumber] = swap;
            } else if (command.equals("multiply")) {
                int firstNumber = Integer.parseInt(commands[1]);
                int secondNumber = Integer.parseInt(commands[2]);

                array[firstNumber] = array[firstNumber] * array[secondNumber];
            } else {
                array = Arrays.stream(array).map(e -> e - 1).toArray();
            }

            input = scan.nextLine();
        }

        // Print result
        System.out.println(Arrays.stream(array).mapToObj(String::valueOf).collect(Collectors.joining(", ")));

        scan.close();
    }
}

File: 24_C#Fundamentals/_03_Arrays/_02_Exercise/_09_ArrayModifier/_09_ArrayModifier.cs
using System;
using System.Linq;

public class _09_ArrayModifier
{
    public static void Main(string[] args)
    {
        // Read input values
        int[] array = Console.ReadLine().Split().Select(int.Parse).ToArray();
        string input = Console.ReadLine();

        // Accept commands and modify the array
        while (!input.Equals("end"))
        {
            string[] commands = input.Split("\\s+");
            string command = commands[0];

            if (command.Equals("swap"))
            {
                int firstNumber = int.Parse(commands[1]);
                int secondNumber = int.Parse(commands[2]);
                int swap = array[firstNumber];

                array[firstNumber] = array[secondNumber];
                array[secondNumber] = swap;
            }
            else if (command.Equals("multiply"))
            {
                int firstNumber = int.Parse(commands[1]);
                int secondNumber = int.Parse(commands[2]);

                array[firstNumber] = array[firstNumber] * array[secondNumber];
            }
            else
            {
                array = array.Select(e => e - 1).ToArray();
            }

            input = Console.ReadLine();
        }

        // Print result
        Console.WriteLine(string.Join(", ", array));
    }
}
